ios/PowerAuth/LiftPowerAuthSdk.m

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2024 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import "PowerAuthObjectRegister.h"
+#import <PowerAuth2/PowerAuthSDK.h>
+#import <React/RCTBridge.h>
+
+PA_EXTERN_C PowerAuthSDK * LiftPowerAuthSdk(NSString * instanceId, RCTBridge * bridge)
```

Comment:
`PA_EXTERN_C` is redundant here. The "extern" keyword is supposed to be used only in headers.